OnBaseDocsApi/App_Start/WebApiConfig.cs

```diff
@@ -15,6 +15,21 @@ public static void Register(HttpConfiguration config)
 
             // Web API routes
             config.MapHttpAttributeRoutes();
+            config.Routes.MapHttpRoute(
+                name: "ApiSecureList",
+                routeTemplate: "api/v1/onbase-docs/Secure",
+                defaults: new { controller = "onbasedocs", action = "Secure" }
+            );
+            config.Routes.MapHttpRoute(
+                name: "ApiSecureGet",
+                routeTemplate: "api/v1/onbase-docs/Secure/{id}",
```

Comment:
Instead of using `{id}`, I recommend rename it with a more descriptive name something like `{hashId}` to avoid confusion since you has already used `{id}` in other endpoints e.g. `api/v1/onbase-docs/{id}`.

Reply:
Good point.